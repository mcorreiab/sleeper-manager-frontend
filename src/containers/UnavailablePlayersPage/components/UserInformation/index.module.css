.userInformation {
  display: flex;
  align-items: center;
  margin-bottom: 1.5rem;
  padding: 0 var(--content-spacing);
}

.userAvatar {
  width: 2rem;
  margin-right: 0.5rem;
}

.username {
  color: #eaeaea;
}
