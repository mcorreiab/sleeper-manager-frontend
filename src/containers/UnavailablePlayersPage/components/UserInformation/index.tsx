import Image from "next/image";
import styles from "./index.module.css";

interface Props {
  username: string;
  avatarUrl: string;
}

const userInformationSection: React.FunctionComponent<Props> = ({
  username,
  avatarUrl,
}) => (
  <div className={styles.userInformation}>
    <div className={styles.userAvatar}>
      <Image src={avatarUrl} alt="User avatar" width={32} height={32} />
    </div>
    <p aria-label="Username" className={styles.username}>
      {username}
    </p>
  </div>
);

export default userInformationSection;
