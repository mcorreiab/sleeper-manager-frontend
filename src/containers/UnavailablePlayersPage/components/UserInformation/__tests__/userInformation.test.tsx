import UserInformation from "../index";
import renderer from "react-test-renderer";

it("Should render user information with John on username", () => {
  const tree = renderer
    .create(<UserInformation username="John" avatarUrl="/avatarUrl" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
