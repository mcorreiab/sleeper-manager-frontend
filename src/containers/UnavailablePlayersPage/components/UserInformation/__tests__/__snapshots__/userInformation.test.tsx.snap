// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render user information with John on username 1`] = `
<div
  className="flex items-center"
>
  <div
    className="w-8 mr-2 h-8"
  >
    <span
      style={
        Object {
          "background": "none",
          "border": 0,
          "boxSizing": "border-box",
          "display": "inline-block",
          "height": "initial",
          "margin": 0,
          "maxWidth": "100%",
          "opacity": 1,
          "overflow": "hidden",
          "padding": 0,
          "position": "relative",
          "width": "initial",
        }
      }
    >
      <span
        style={
          Object {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "padding": 0,
            "width": "initial",
          }
        }
      >
        <img
          alt=""
          aria-hidden={true}
          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2732%27%20height=%2732%27/%3e"
          style={
            Object {
              "background": "none",
              "border": 0,
              "display": "block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "padding": 0,
              "width": "initial",
            }
          }
        />
      </span>
      <img
        alt="User avatar"
        data-nimg="intrinsic"
        decoding="async"
        onError={[Function]}
        onLoad={[Function]}
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
      <noscript>
        <img
          alt="User avatar"
          data-nimg="intrinsic"
          decoding="async"
          loading="lazy"
          src="/_next/image?url=%2FavatarUrl&w=64&q=75"
          srcSet="/_next/image?url=%2FavatarUrl&w=32&q=75 1x, /_next/image?url=%2FavatarUrl&w=64&q=75 2x"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
      </noscript>
    </span>
  </div>
  <p
    aria-label="Username"
    className="text-[#eaeaea] font-bold"
  >
    John
  </p>
</div>
`;
